src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,279 @@
+                        'for stills a pdf. Name of file should not include '
+                        'file extension. If not specified, renders to screen.')
+    group_anim = parser.add_mutually_exclusive_group(required=False)
+    group_anim.add_argument(
+        '--fps',
+        type=float,
+        default=5,
+        help='Set the number of frames per second when writing '
+        'an animation to disk.')
+    group_anim.add_argument('--interval',
+                            type=float,
+                            help='Delay between frames in  milliseconds')
+    # Check for h5 suffix if '--file-prefix' argument used
+    args = parser.parse_args()
+    assert args.file_prefix is None or not args.file_prefix.endswith('.h5'),\
+    "Retry without .h5 extension on 'file-prefix'"
+
+    return vars(args)
+
+
+def get_h5_files(files):
+    '''
+    Get a list of the h5 files containing the data
+    :return list of h5 files containing volume data
+    '''
+
+    h5_files = []
+    if isinstance(files, str):
+        h5_file_names = glob.glob(files + "*.h5")
+        assert len(h5_file_names) > 0,\
+            "Found no files with prefix {} in directory {}"\
+            .format(files, os.getcwd())
+        h5_files = [h5py.File(file_name, 'r') for file_name in h5_file_names]
+
+    else:
+        for file_name in files:
+            try:
+                h5_files.append(h5py.File(file_name, 'r'))
+            except IOError as err:
+                sys.exit("Could not find file '{}' in directory {}: {}"\
+                         .format(file_name, os.getcwd(), err))
+    return h5_files
+
+
+def print_var_names(files):
+    '''
+    Print all available variables to screen
+    :param args:
+    :return: None
+    '''
+
+    h5files = get_h5_files(files)
+    volfile = h5files[0]["element_data.vol"]
+    obs_id_0 = next(iter(volfile))
+    variables = list(volfile[obs_id_0].keys())
+    variables.remove("connectivity")
+    variables.remove("InertialCoordinates_x")
+    variables.remove("total_extents")
+    variables.remove("grid_names")
+    print("Variables in H5 file:\n{}".format(list(map(str, variables))))
+    for h5_file in h5files:
+        h5_file.close()
+    return None
+
+
+def get_data(files, var_name):
+    '''
+    Get the data to be plotted
+    :return: the array of data, coords, and time
+    '''
+
+    time = []
+    coords = []
+    data = []
+    h5files = get_h5_files(files)
+    volfiles = [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id, volfiles[0][obs_id].attrs['observation_value'])
+                 for obs_id in volfiles[0].keys()]
+    ids_times.sort(key=lambda pair: pair[1])
+    for obs_id, local_time in ids_times:
+        local_coords = []
+        local_data = []
+        ordering = None
+        for volfile in volfiles:
+            try:
+                local_data = local_data +\
```

Comment:
FYI, the trick with the parentheses to break long lines works also for operator expressions like this